pkg/cmd/repo/rename/rename.go

```diff
@@ -144,6 +148,21 @@ func renameRun(opts *RenameOptions) error {
 		fmt.Fprintf(opts.IO.Out, "%s Updated the %q remote\n", cs.SuccessIcon(), remote.Name)
 	}
 
+	if opts.DoConfirm && !opts.RenameLocalDir {
+		var confirmed bool
+		if confirmed, err = opts.Prompter.Confirm("Would you also like to rename the repo directory to ?", false); err != nil {
+			return err
+		}
+		if confirmed {
+			opts.RenameLocalDir = true
+		}
+	}
+	if opts.RenameLocalDir {
```

Comment:
We should probably check somewhere that the name the user has chosen isn't already in-use. We should probably do this before the API call, even, and can error if it is.